portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,53 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+body {
+    background-color: white;
+    color: black;
+}
+
+h1 {
+    text-align: center;
+    background-color:lightgray;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+footer {
+    text-align: center;
+    background-color: black;
+    color: white;
+}
+
+.box-layout {
+    display: inline-block;
+    width: 400px;
+    height: 100px;
+    margin: 1px;
+    border: 1px;
+}
+
+h3 {
+    text-align: center;
+}
+
+.button {
```

Comment:
nit: maybe rename to hyperlinked-button? It will be clear that it's a [ CSS class selector](https://www.w3schools.com/cssref/sel_class.asp) not a [tag](https://www.w3schools.com/html/html_css.asp) applicable to all html <button> elements.